README.md

```diff
@@ -203,6 +238,45 @@ To evaluate a finetuned model, use the following command from the project basedi
 - [Pretrained ImageNet weights](https://github.com/amirbar/DETReg/releases/download/1.0.0/checkpoint_imagenet.pth) 
 - [Finetuned COCO weights](https://github.com/amirbar/DETReg/releases/download/1.0.0/full_coco_finetune.pth)
 
+### Using hubconfig for evaluation for DETReg with MDef-DETR proposals:
+1. Pretrained model with MDef-DETR proposals:
+```bash
+checkpoints_path = "./exps/DETReg_mdef_detr_prop_top30_in100/checkpoint.pth"
+model = deformable_detr_resnet50(pretrained=False, return_postprocessor=False, num_classes=91,
+                                 object_embedding_loss=True, checkpoints_path=checkpoints_path)
+```
+2. Fine tune on 10% of Pascal:
+```bash
+checkpoints_path = "./exps/DETReg_mdef_detr_prop_fine_tune_10pct_pascal/checkpoint.pth"
+model = deformable_detr_resnet50(pretrained=False, return_postprocessor=False, num_classes=21,
+                                 object_embedding_loss=False, checkpoints_path=checkpoints_path)
+```
+3. Fine tune on full of Pascal:
+```bash
+checkpoints_path = "./exps/DETReg_mdef_detr_prop_fine_tune_full_pascal/checkpoint.pth"
+model = deformable_detr_resnet50(pretrained=False, return_postprocessor=False, num_classes=21,
+                                 object_embedding_loss=False, checkpoints_path=checkpoints_path)
+```
+
+### Using hubconfig for evaluation for DETReg with MDETR proposals:
+1. Pretrained model with MDETR proposals:
+```bash
+checkpoints_path = "./exps/DETReg_mdetr_prop_top30_in100/checkpoint.pth"
+model = deformable_detr_resnet50(pretrained=False, return_postprocessor=False, num_classes=91,
+                                 object_embedding_loss=True, checkpoints_path=checkpoints_path)
+```
+2. Fine tune on 10% of Pascal:
+```bash
+checkpoints_path = "./exps/DETReg_mdetr_prop_fine_tune_10pct_pascal/checkpoint.pth"
+model = deformable_detr_resnet50(pretrained=False, return_postprocessor=False, num_classes=21,
+                                 object_embedding_loss=False, checkpoints_path=checkpoints_path)
+```
+3. Fine tune on full of Pascal:
+```bash
+checkpoints_path = "./exps/DETReg_mdetr_prop_fine_tune_full_pascal/checkpoint.pth"
+model = deformable_detr_resnet50(pretrained=False, return_postprocessor=False, num_classes=21,
+                                 object_embedding_loss=False, checkpoints_path=checkpoints_path)
+```
```

Comment:
I see for both MDETR & MDef-DETR, only the checkpoints path is different. If so, we can mention pointing `checkpoints_path` to the relevant model and avoid the duplicate instructions

Reply:
Yeah sure.